draft-davidben-tls-trust-expr.md

```diff
@@ -247,7 +253,7 @@ A trust store manifest is a JSON object {{!RFC8259}} containing:
   `entries`:
   : A non-empty array describing the contents of this version of the trust store. These are known as trust store entries. Each element is an object containing:
 
-    `id`:
+    `trust_anchor`:
     : A string containing the name of some member of the `trust_anchors` object.
```

Comment:
in text form as desribed in {{trust-store-ids}}}

Reply:
Oh, this isn't an OID. Those are used to name the trust stores. This is an index into the `trust_anchors` object. (That part of the design also maybe needs work, but we're not doing anything with it just yet.)